scripts/background/Utility.js

```diff
@@ -6,13 +6,13 @@ var DEBUG_FLAG = false;
 
 // format text for the badge and for the song URL
 function formatHour(time) {
-	if (time === -1) {
+	if (time == -1) {
```

Comment:
What's the reason for this change? I guess we are sending `time` as a string somewhere?  Damn javascript and it's equality checks

Reply:
This was an issue that occurred during testing, it's not really needed, however was done to prevent any future issues as strict equality checks should only be used when necessary.